docs: Document SAML password hiding and login retry screen (#819)
mautic/user-documentation: docs/users_roles/managing_users.rst

Reviewer: The indentations here needs to be updated too.

```suggestion
   * **Time zone** - Set the User's time zone, or use the default. Adding the User's time zone enables them to account for time zone differences for Email scheduling and other features.
   * **Language** - Select a language for each User, to improve their experience in Mautic. 
```

Apply this to all other PRs.

   * **Position - optional** - Your User's job title.
  
   * **Username, Email, Password** are the login credentials. If a User forgets their password, they can use the Forgot password link, but you can manually change their password here.

>      Passwords must be at least six characters in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.

Reviewer: ```suggestion
     Passwords must be **at least six characters** in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.
```

   * **Position - optional** - Your User's job title.
  
   * **Username, Email, Password** are the login credentials. If a User forgets their password, they can use the Forgot password link, but you can manually change their password here.

     Passwords must be **at least six characters** in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.

     .. note::

        .. vale off

        When you log in through SAML, Mautic hides the password fields on the User add and edit forms. SAML Users manage their passwords in the identity provider, not in Mautic. For more information, see :doc:`SAML Single Sign On </authentication/authentication>`.

        .. vale on

   * **Time zone** - Set the User's time zone, or use the default. Adding the User's time zone enables them to account for time zone differences for Email scheduling and other features.

   * **Language** - Select a language for each User, to improve their experience in Mautic.

> When creating your Users, inform them of the credentials. Mautic doesn't send an Email notifying Users of their login information. Manually informing them is necessary.

Reviewer: I feel like this is an important information for admins.

```suggestion
.. important::

   .. vale off

   Once you create Users, give them their credentials directly because Mautic doesn't send emails with their login information.

   .. vale on
```

   * **Position - optional** - Your User's job title.
  
   * **Username, Email, Password** are the login credentials. If a User forgets their password, they can use the Forgot password link, but you can manually change their password here.

     Passwords must be **at least six characters** in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.

     .. note::

        .. vale off

        When you log in through SAML, Mautic hides the password fields on the User add and edit forms. SAML Users manage their passwords in the identity provider, not in Mautic. For more information, see :doc:`SAML Single Sign On </authentication/authentication>`.

        .. vale on

   * **Time zone** - Set the User's time zone, or use the default. Adding the User's time zone enables them to account for time zone differences for Email scheduling and other features.

   * **Language** - Select a language for each User, to improve their experience in Mautic.

.. important::

   .. vale off

   Once you create Users, give them their credentials directly because Mautic doesn't send emails with their login information.

>    .. vale on

Reviewer: Excluding the note admonition, apply the same changes to https://github.com/mautic/user-documentation/pull/823 and https://github.com/mautic/user-documentation/pull/824

  
   * **Username, Email, Password** are the login credentials. If a User forgets their password, they can use the Forgot password link, but you can manually change their password here.
  
.. note::

    Passwords must be at least six characters in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.

.. note::

>    When you log in through SAML, Mautic hides the password fields on the User add and edit forms. SAML Users manage their passwords in the identity provider, not in Mautic. For more information, see :doc:`SAML Single Sign On </authentication/authentication>`.

Author: Added the note that SAML users see no password fields on the User add/edit forms because PR #15508's `User/form.html.twig` guards `form.plainPassword` with `{% if isSamlUser is empty %}` and `UserController::editAction` removes `plainPassword` when `samlHelper->isSamlSession()` is true.

Source: https://github.com/mautic/mautic/pull/15508

>    When you log in through SAML, Mautic hides the password fields on the User add and edit forms. SAML Users manage their passwords in the identity provider, not in Mautic. For more information, see :doc:`SAML Single Sign On </authentication/authentication>`.

Reviewer: ```suggestion
   .. vale off
   
   When you log in through SAML, Mautic hides the password fields on the User add and edit forms. SAML Users manage their passwords in the identity provider, not in Mautic. For more information, see :doc:`SAML Single Sign On </authentication/authentication>`.
   
   .. vale on
```

  
   * **Username, Email, Password** are the login credentials. If a User forgets their password, they can use the Forgot password link, but you can manually change their password here.
  
.. note::

>     Passwords must be at least six characters in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.

Reviewer: Having a back to back admonitions can be confusing.

```suggestion
   * **Username, Email, Password** are the login credentials. If a User forgets their password, they can use the Forgot password link, but you can manually change their password here.

     Passwords must be at least six characters in length. Ensure that you use a combination of upper and lower case alphabets, special characters, and numbers.
```

Apply this to all cherry pick PRs.

.. note::

   .. vale off

   When you log in through SAML, Mautic hides the password fields on the User add and edit forms. SAML Users manage their passwords in the identity provider, not in Mautic. For more information, see :doc:`SAML Single Sign On </authentication/authentication>`.

>    .. vale on

Reviewer: Fix the indentation as this is part of a bullet point.

```suggestion
  .. note::

     .. vale off

     When you log in through SAML, Mautic hides the password fields on the User add and edit forms. SAML Users manage their passwords in the identity provider, not in Mautic. For more information, see :doc:`SAML Single Sign On </authentication/authentication>`.

     .. vale on
```

Apply this to https://github.com/mautic/user-documentation/pull/825 and https://github.com/mautic/user-documentation/pull/826
